runtimes.json

```diff
@@ -4,13 +4,13 @@
         "0.3.DL": "0.3.DL-R1",
         "0.3.7": "0.3.7-R2-2-1",
         "0.3z": "0.3z-R4",
-        "UG-MP": "UGMP"
+        "UGMP": "UG-MP"
```

Comment:
It would be good to version these in future.

Reply:
As of the time being, they do not have versioning. IIRC, their server files are just 0.3.7 R1, and they don't have permission to use anything other than that, so they'll likely be sticking with that.